2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,13 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
```

Comment:
Looks perfect!
One small point - just keep an eye on indentation (spacing). This will make it easier for other developers to read your code.

Reply:
thanks, i will fix it .